src/pages/messages/components/TicketList.tsx

```diff
@@ -36,6 +39,7 @@ export const TicketList = ({ ticketsQuery, setRowsPerPage, rowsPerPage, setPage,
   const { t } = useTranslation();
   const history = useHistory();
   const isOnTasksPage = history.location.pathname === UrlPathTemplate.TasksDialogue;
+  const user = useSelector((store: RootState) => store.user);
```

Comment:
kunne muligens bare sende denne som prop fra parents?

Reply:
Jeg ville nok foretrukket `useSelector`, siden den er der for å unngå prop-drilling. Samme med alle forekomster vi har av `useFormikContext`, `useSelector` og de få `useContext` vi selv har lagt inn. Vi kunne henta `user` bare i den øverste komponenten og sendt den som prop til alle komponenter under, men det ønsker vi ikke, da det er my ryddigere å bare hente den inn der den faktisk trengs, så trenger ikke parent å vite noe om det